frontend/src/metabase/plugins/index.ts

```diff
@@ -128,6 +128,8 @@ export const PLUGIN_SELECTORS = {
   canWhitelabel: (_state: State) => false,
   getLoadingMessage: (_state: State) => t`Doing science...`,
   getIsWhiteLabeling: (_state: State) => false,
+  // Don't need to wrap this in `ttag` because we're not using ttag to change whitelabel name anymore
+  getApplicationName: (_state: State) => "Metabase",
```

Comment:
Don't need to wrap this in `ttag` because we're not using ttag to change whitelabel name anymore